interpreter/cling/lib/Interpreter/ClingPragmas.cpp

```diff
@@ -22,19 +22,32 @@ using namespace cling;
 using namespace clang;
 
 namespace {
-  class PHLoad: public PragmaHandler {
-    Interpreter& m_Interp;
+   static void replaceEnvVars(std::string &Path){
 
-  public:
-    PHLoad(Interpreter& interp):
-      PragmaHandler("load"), m_Interp(interp) {}
+    std::size_t fpos = Path.find("$");
 
-    void HandlePragma(Preprocessor &PP,
-                      PragmaIntroducerKind Introducer,
-                      Token &FirstToken) override {
-      // TODO: use Diagnostics!
+    while (fpos != std::string::npos) {
+      std::size_t spos = Path.find("/", fpos + 1);
+      std::size_t length;
+
+      if (spos != std::string::npos) // if we found a "/"
+        length = spos - fpos;
+      else // we didn't find any "/"
+        length = Path.length();
 
-      struct SkipToEOD_t {
+      std::string envVar = Path.substr(fpos + 1, length -1); //"HOME"
+      std::string fullPath{getenv(envVar.c_str())};
+      Path.replace(fpos, length, fullPath);
+      fpos = Path.find("$", fpos + 1); //search for next env variable
+    }
+  }
+  static std::string HandlePragmaHelper(Preprocessor &PP,
+                      PragmaIntroducerKind Introducer,
+                      Token &FirstToken,
+                      Interpreter &m_Interp,
+                      std::string errorMessage,
+                      std::string pragmaInst){
```

Comment:
should both be `const std::string &` to not copy the string (even though C++ move ctors might help but...)